### ISSUE TITLE
Prefilled Reddit-post shows up empty.

### ISSUE THREAD

**Issue by Steinrik** [keybase/keybase-issues#2935]
I've tried on both the latest Opera and Chrome browsers on my Samsung Galaxy S6 with stock Android 6.01.
1: log into Reddit. 
2: clicked to open the prefilled Reddit post.
3: post opens with empty text fields.

The end...

**Comment by cjb** [comment]
Thanks for the report, CC @oconnor663 @mlsteele

**Comment by oconnor663** [comment]
I noticed that the new reddit mobile site isn't respecting the pre-fill parameters. Is this what you say, @Steinrik?

![image](https://cloud.githubusercontent.com/assets/860932/24812330/9dc789c0-1b97-11e7-9e72-85aaa87fe7f9.png)

**Comment by Steinrik** [comment]
Yes. I've tried several browsers, am starting to blame Reddit itself.

Mvh
Stein-Erik Dahle

7. apr. 2017 19:40 skrev "oconnor663" <notifications@github.com>:
